standard.md

```diff
@@ -0,0 +1,134 @@
+# Definition of the standard
+
+This document intends to define a standard
+with the purpose of facilitating the sharing of source codes used to
+analyze climate data and produce evaluation results and figures. 
+Said codes will hereafter be referred as 'diagnostic scripts', or in short 'diagnostics'
+or 'scripts'. 
+
+It is assumed that diagnostic scripts are able to be wrapped
+as standalone codes that run as independent processes, an approach that
+has been successfully tested in tools such as ESMValTool and CLiMAF.
+It also assumes that diagnostic scripts are able to read CF compliant NetCDF
+data files and to produce output files with formats such as NetCDF, CVS, txt,
+or excel spreadsheets; as well as graphic files.
+
+## Glossary
+
+This standard defines three tiers of enforceability for the guidelines to share diagnostics:
+
+- **Must**: guidelines that all diagnostics have to meet to comply with the standard.
+- **Recommend**: guidelines that all diagnostics are encouraged to meet, but are not mandatory.
+Nevertheless, if a list of compatible diagnostics is compiled, diagnostic scripts meeting
+recommended guidelines will be highlighted.
+- **Suggest**: guidelines that are not required, but are considered to improve the quality of the diagnostics.
+
+Diagnostic scripts are expected to be contributed along with a YAML file containing configuration options.
```

Comment:
I do not understand this sentence. I would understand that scripts are contributed with a **formal description file**. But the configuration file described below is not a contributed one. And the enforceability levels defined just below apply to the configuration file

Reply:
It has been like this since the first commit. The formal description file is considered a recommended contribution to complete this configuration file.

Reply:
Yes. What I mean is that it is ambiguous to write that the description file contains *configuration** options while, in the same time, naming another file the **configuration** file

Reply:
Thanks for the explanation, I removed any mention of configuration options in the fromal description file section. Hope that works for you.